.env_vars.sample

```diff
@@ -1,9 +1,11 @@
 #next
 export CONNECT_ACCOUNT_PORT=***
 export CONNECT_ACCOUNT_SESSION_SALT=***
+export CONNECT_ACCOUNT_DOMAIN=***
```

Comment:
check that it works in preview with `CONNECT_ACCOUNT_DOMAIN=https${VERCEL_URL}`

Reply:
Ok, I really don't get how this could work on dev env, as you told me not to do a ternary check, along the lines of

```ts
  if (!process.env.CONNECT_ACCOUNT_DOMAIN) {
    console.log("Missing CONNECT_ACCOUNT_DOMAIN");
    process.exit(1);
  }

  if (process.env.VERCEL_URL) {
    config.connectDomain = process.env.CONNECT_ACCOUNT_DOMAIN;
  } else {
    config.connectDomain = "http://localhost:29703";
  }
```

Am I missing something?